plugin/cql/src/main/java/org/opencds/cqf/ruler/cql/KnowledgeArtifactProcessor.java

```diff
@@ -574,6 +584,252 @@ private Optional<MetadataResource> checkIfReferenceInList(RelatedArtifact artifa
 		}
 		return updatedReference;
 	}
+	/* $package */
+	public Bundle createPackageBundle(IdType id, FhirDal fhirDal, List<String> capability, List<String> include, List<CanonicalType> canonicalVersion, List<CanonicalType> checkCanonicalVersion, List<CanonicalType> forceCanonicalVersion, Integer count, Integer offset, Endpoint contentEndpoint, Endpoint terminologyEndpoint, Boolean packageOnly){
+		if (contentEndpoint != null || terminologyEndpoint != null) {
+			throw new NotImplementedOperationException("This repository is not implementing custom Content and Terminology endpoints at this time");
+		}
+		if (packageOnly != null) {
+			throw new NotImplementedOperationException("This repository is not implementing packageOnly at this time");
+		}
```

Comment:
I haven't implemented packageOnly since I don't know how to get to NpmPackageManager, If it's easy I can add it to this PR otherwise maybe we move it to an enhancement for later?

Similarly for contentEndpoint and terminologyEndpoint are they useful for MVP or are we better off adding them as enhancements?

Reply:
I think this is fine for MVP.